### ISSUE TITLE
[React Native][Thirdweb v5] Error trying to implement SIWE on Expo API routes

### ISSUE THREAD

**Issue by ludwig-g-w** [thirdweb-dev/js#3299]
It seems like the error arrives when using `import { privateKeyToAccount } from "thirdweb/wallets"` on the server side, if I remove it it succeeds to build.

The error I get:

```
node_modules/thirdweb/dist/cjs/wallets/wallet-connect/controller.js: /my-project/node_modules/thirdweb/dist/cjs/wallets/wallet-connect/controller.js: Duplicate declaration "Linking" (This is an error on an internal node. Probably an internal error.)
```

here is the code that fails:

```
import { createThirdwebClient } from "thirdweb";
import { createAuth } from "thirdweb/auth";
import { privateKeyToAccount } from "thirdweb/wallets";

export const thirdwebClient = createThirdwebClient({
  secretKey: process.env.EXPO_PUBLIC_THIRDWEB_AUTH_PRIVATE_KEY,
});

export const thirdwebAuth = createAuth({
  domain: process.env.EXPO_PUBLIC_SERVER_URL || "",
  client: thirdwebClient,
  adminAccount: privateKeyToAccount({
    client: thirdwebClient,
    privateKey: process.env.EXPO_PUBLIC_THIRDWEB_AUTH_PRIVATE_KEY!,
  }),
});
```

**Comment by jnsdls** [comment]
excited about expo api routes! we'll take a look at this ASAP and should be able to make it work!

(we haven't played with the new expo router stuff yet, here's an excuse to do so!)

**Comment by joaquim-verges** [comment]
@ludwig-g-w we just did a pretty significant refactor on the way we do react native exports.

Could you try installing latest nightly and let us know if it solves this issue for you? 

`npm i thirdweb@nightly`

**Comment by ludwig-g-w** [comment]
Cool I confirm that it's not crashing anymore 🙏

**Comment by jnsdls** [comment]
the stable release will be out early next week, we'll close this issue once its out - thanks for helping us make it better!

**Comment by jnsdls** [comment]
fixed as part of: [5.29.3](https://github.com/thirdweb-dev/js/releases/tag/thirdweb%405.29.3)
